client/config/index.js

```diff
@@ -12,8 +12,4 @@ if ( 'undefined' === typeof window ||  ! window.config ) {
  * @property {Function} anyEnabled whether one of a set of properties is enabled
  * @property {Function} isEnabled whether a specific property is enabled
  */
-const config = window.config;
-config.isEnabled = window.isEnabled;
-config.anyEnabled = window.anyEnabled;
-
-export default config;
+export default require( '../../server/config' );
```

Comment:
I'm not very familiar with isomorphic code best practices:
> I don't think we should ever reference server code inside client

whats the reason for that?